.github/workflows/qa_build.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
+          pip install zbar-py==1.0.4
```

Comment:
Should we not be adding this to `setup.py` and `requirements.txt`, instead of installing it manually?